Tasks/src/Stringarrayfilter.java

```diff
@@ -0,0 +1,92 @@
+import java.util.Scanner;
+
+public class Stringarrayfilter {
+
+    public static void main(String[] args) {
+
+        final String result = "Результат вашего запроса следующий :";
+        Scanner sc = new Scanner(System.in);
+        byte more;
+        int zero;
+        System.out.println("Введите данные в формате [name1, name2, ..., nameX]");
+        String str = sc.nextLine();
+        String[] words = str.split(", ");
+        boolean check = true;
+        while (check) {
+            System.out.println("Какой фильтр желаете применить для данных?");
+            System.out.println("1 | Максимальная длинна");
+            System.out.println("2 | Начальная буква(символ)");
+            System.out.println("3 | Конечная буква(символ)");
+            System.out.println("4 | Содержание");
```

Comment:
Please rewrite to more understandable name of function (what it should do)

Reply:
Done